### PR TITLE
tsdb: accept millisecond-min-interval

### PR DESCRIPTION
**Pull request by gabor**
when in a dashboard-panel, i have a prometheus/influxdb-influxql/postgres query, and add an Expression (operation: Math, Expression: `$A` ... so it just returns the query-result), the min-interval query-option does not get applied.

it seems the code they use to get the interval-value is picking data from a JSON structure, and it picks the string-field named "interval". for the datasources i mentioned the value is an int64-field named "intervalMs".

so i created this diff that first tries the "intervalMs". it works, but i'm not sure if this is the right approach. i don't even know which data source uses the string "interval" field. 
also, this code is used in other datasources too, like `azuremonitor`, `elastic`, `sqleng`, where i did not test it, so i don't know if this breaks anything.

fixes https://github.com/grafana/grafana/pull/34597

### CODE REVIEW

File: pkg/tsdb/interval/interval.go [hunk 1]
```diff
@@ -67,6 +67,11 @@ func (ic *intervalCalculator) Calculate(timerange plugins.DataTimeRange, minInte
 }
 
 func GetIntervalFrom(dsInfo *models.DataSource, queryModel *simplejson.Json, defaultInterval time.Duration) (time.Duration, error) {
+	intervalMs := queryModel.Get("intervalMs").MustInt64(-1)
+	if intervalMs != -1 {
+		return time.Millisecond * time.Duration(intervalMs), nil
```
Comment (aknuds1) [inline pkg/tsdb/interval/interval.go:72]: Just a nit, the standard in Go is [consistent case for initialisms](https://github.com/golang/go/wiki/CodeReviewComments#initialisms):
```suggestion
	intervalMS := queryModel.Get("intervalMs").MustInt64(-1)
	if intervalMS != -1 {
		return time.Millisecond * time.Duration(intervalMS), nil
```

**Review by aknuds1 — COMMENTED** [review]
Technically it looks fine to me, but I'm not the right person to tell if this is the right place to fix this or not since I'm unfamiliar with the functionality. Maybe @marefr or @bergquist know?

**Comment by marefr** [comment]
Looks like a duplicate of #34270. Please have a look at that PR and whether discussions and changes make sense to you.

**Comment by gabor** [comment]
Duplicate of https://github.com/grafana/grafana/pull/34270
closing.

**Pull request closed without merging**